### DISCUSSION TITLE
Inquiry about Support and Testing for Aurora Version 2 (MySQL 5.7)

### DISCUSSION THREAD

**Discussion by cprayer** [aws/aws-advanced-jdbc-wrapper#815] [Q&A] [answered]
Hello AWS Labs Team,

I am writing to inquire about the support and testing for AWS Aurora MySQL Version 2 (compatible with MySQL 5.7) in the aws-advanced-jdbc-wrapper project. In my organization, we are actively using Aurora Version 2

I have observed that the current testing efforts for the aws-jdbc-advanced-wrapper are mainly focused on Aurora 3 (MySQL 8). This prompts a question about the level of support and compatibility for Aurora Version 2

Understanding the compatibility and support for Aurora Version 2 in relation to the aws-advanced-jdbc-wrapper is essential for us, as it directly impacts our integration and operational decisions.

Could you please provide information on:

The current extent of testing and support for Aurora Version 2 (MySQL 5.7) within the aws-advanced-jdbc-wrapper project.
Any plans to expand support and testing to include Aurora Version 2, if it is not currently included.
Thank you for your assistance and the work you do in maintaining this project.

Best regards,

**Comment by davecramer** [comment, marked as answer]
We do not currently test against Aurora MySQL version 2. 
That said; since we are wrapping the native mysql driver I would expect the driver to work.
Have you tested it ? Are there any issues with it ?

**Reply by cprayer** [reply]
@davecramer 
I missed your response and am replying late. I apologize for that.
When tested on AWS Aurora 2 (MySQL 5.x version), there were no issues.
If the information_schema structure is the same in both Aurora 2 and 3, there shouldn't be any problems.
